### PR TITLE
adds documentation for configuring github repo discovery

### PR DESCRIPTION
**Pull request by punkle**
[ch1968]

### CODE REVIEW

**Review by Xantier — APPROVED** [review]

File: content/docs/getting-started/adding-components/index.md [hunk 1]
```diff
@@ -64,6 +64,18 @@ Click the Home link in the Backstage sidebar to go back to the catalog where you
 
 ![our component visible on the other tab of Backstage](./component-on-other-tab.png)
 
+## Configuring Backstage to auto discover github repositories using the github discovery processor
+
+The Roadie team can configure your backstage instance to auto discover catalog yaml files created in your github repositories. To do this, please contact one of us and we will be happy to configure it for you.
```
Comment (dtuite) [inline content/docs/getting-started/adding-components/index.md:69]:
```suggestion
The Roadie team can configure your Backstage instance to auto discover `catalog-info.yaml` files created in your GitHub repositories. To do this, please contact us via Slack and we will be happy to configure it for you.
```

Fixed some capitalization issues. `catalog-info.yaml` is the canonical name of the files so you can use that exact term.

File: content/docs/getting-started/adding-components/index.md [hunk 2]
```diff
@@ -64,6 +64,18 @@ Click the Home link in the Backstage sidebar to go back to the catalog where you
...
+
+We will need a url pattern to discover the catalog files.
```
Comment (dtuite) [inline content/docs/getting-started/adding-components/index.md:71]:
```suggestion
We will need a URL pattern to discover the catalog files.
```

File: content/docs/getting-started/adding-components/index.md [hunk 3]
```diff
@@ -64,6 +64,18 @@ Click the Home link in the Backstage sidebar to go back to the catalog where you
...
+
+```text
+https://github.com/AcmeInc/*/blob/main/catalog-info.yaml
+```
+
+You can use regular regex syntax in the repository name part of the github url as shown above.
```
Comment (dtuite) [inline content/docs/getting-started/adding-components/index.md:77]:
```suggestion
You can use regular regex syntax in the repository name part of the GitHub URL as shown above.
```

**Review by dtuite — CHANGES_REQUESTED** [review]
Mostly just capitalization fixes.

**Review by dtuite — APPROVED** [review]

**Pull request merged**